server/server.go

```diff
@@ -1960,6 +1973,27 @@ func (s *Server) AcceptLoop(clr chan struct{}) {
 	clr = nil
 }
 
+// InProcessConn returns a connection to the server which can
+// be used in-process when DontListen is set.
```

Comment:
This seem to be usable even when DontListen is not set though right, that's good I'd be keen on this feature also but I want a mix or listening sockets and inprocess connections. So maybe comment needs an update?

Reply:
It is indeed usable either way, so I'll update the comment!